Fixed zonedTimeToUtc not working with year < 100.

make-github-pseudonymous-again opened the pull request:
Fixes #87.

make-github-pseudonymous-again commented on src/zonedTimeToUtc/index.js:
```diff
@@ -36,15 +36,11 @@ export default function zonedTimeToUtc(date, timeZone, options) {
-    d.getSeconds(),
-    d.getMilliseconds()
-  )
+  var tmp = new Date(0)
+  tmp.setUTCFullYear(d.getFullYear(), d.getMonth(), d.getDate())
+  tmp.setUTCHours(d.getHours(), d.getMinutes(), d.getSeconds(), d.getMilliseconds())
+
+  var utc = tmp.getTime()
```
Refactor using new helper functions in `tzParseTimezone`.

make-github-pseudonymous-again commented on src/_lib/tzParseTimezone/index.js:
```diff
@@ -70,23 +70,32 @@ export default function tzParseTimezone(timezoneString, date, isUtcDate) {
   var tokens = tzTokenizeDate(date, timezoneString)
 
-  var asUTC = Date.UTC(tokens[0], tokens[1] - 1, tokens[2], tokens[3] % 24, tokens[4], tokens[5])
+  var asUTC = DateUTC(tokens[0], tokens[1] - 1, tokens[2], tokens[3] % 24, tokens[4], tokens[5], 0)
```
It seems milliseconds are dropped here, intentional? I can write a second PR to fix that.

make-github-pseudonymous-again commented on src/_lib/tzTokenizeDate/test.js:
```diff
@@ -20,6 +20,11 @@ describe('tzTokenizeDate', function () {
 
+  it('works with year < 100', function () {
+    var result = tzTokenizeDate(new Date('0099-01-01T00:00:00.000Z'), 'UTC')
+    assert.deepEqual(result, [99, 1, 1, 0, 0, 0, 0])
```
Oops the last zero needs to be gone for now, please review changes first, I'll force push later.

make-github-pseudonymous-again replied:
Done!

make-github-pseudonymous-again reviewed (commented):
I do not think linting was run on this commit. `eslint` barfs something about a deprecation warning:
```console
(node:620180) [ESLINT_PERSONAL_CONFIG_SUPPRESS] DeprecationWarning: '~/.eslintrc.*' config files have been deprecated. Please remove it or add 'root:true' to the config files in your projects in order to avoid loading '~/.eslintrc.*' accidentally. (found in "../../../.eslintrc")
(Use `node --trace-deprecation ...` to show where the warning was created)
```

Did I do something wrong?

marnusw commented:
@make-github-pseudonymous-again thank you for working on this. I'd be happy to merge this in once completed and the conflicts resolved.

make-github-pseudonymous-again commented:
> @make-github-pseudonymous-again thank you for working on this. I'd be happy to merge this in once completed and the conflicts resolved.

@marnusw I rebased and fixed the test with the extra trailing zero. What about linting? I think the other comments I made can be addressed in a different PR.

marnusw commented:
Released in `v1.3.2`.
